cl/simple_pages/templates/help/relative_dates_help.html

```diff
@@ -0,0 +1,75 @@
+{% extends "base.html" %}
+
+{% block title %}Help with Relative Dates – CourtListener.com{% endblock %}
+{% block og_title %}Help with Relative Dates – CourtListener.com{% endblock %}
+
+{% block description %}Learn how to use relative dates in CourtListener to make your searches and alerts more dynamic and up-to-date.{% endblock %}
+{% block og_description %}Learn how to use relative dates in CourtListener to make your searches and alerts more dynamic and up-to-date.{% endblock %}
+
+{% block sidebar %}{% endblock %}
+
+{% block footer-scripts %}
+  {% include "includes/anchors.html" %}
+{% endblock %}
+
+{% block content %}
+<div class="col-xs-12 hidden-md hidden-lg">
+  <h4 class="v-offset-below-2">
+    <i class="fa fa-arrow-circle-o-left gray"></i>
+    <a href="{% url "help_home" %}">Back to Help</a>
+  </h4>
+</div>
+
+<div id="toc-container" class="hidden-xs hidden-sm col-md-3">
+  <div id="toc">
+    <h4 class="v-offset-below-3">
+      <i class="fa fa-arrow-circle-o-left gray"></i>
+      <a href="{% url "help_home" %}">Back to Help</a>
+    </h4>
+    <h3>Table of Contents</h3>
+    <ul>
+      <li><a href="#relative-dates">Relative Dates</a></li>
+      <li><a href="#syntax">Allowed Syntax</a></li>
+      <li><a href="#notes">Technical Notes</a></li>
+    </ul>
+  </div>
+</div>
+
+<div class="col-xs-12 col-md-8 col-lg-6">
+  <h1 id="relative-dates">Help with Relative Dates</h1>
+  <p class="lead">Use relative dates in your queries to keep your searches and alerts dynamically up to date.</p>
+  <p>When placing a search query into CourtListener, you can use either “Calendar” date queries or “Relative” date queries:</p>
+  <ul>
+    <li>Calendar date queries let you filter before or after a given date.</li>
+    <li>Relative dates filter results dynamically based on the date the query is made by a user or system. Instead of specifying a calendar date, such a “January 1, 2025,” you can use relative time periods like <code>1 day ago</code>, <code>Past 1 month</code>, or <code>-14d</code>.</li>
```

Comment:
```suggestion
    <li>Relative dates filter results dynamically based on the date the query is made by a user or system. Instead of specifying a calendar date, such a "January 1, 2025," you can use relative time periods like <code>1 day ago</code>, <code>Past 1 month</code>, or <code>-14d</code>.</li>
```